src/modules/navigator/mission_base.cpp

```diff
@@ -235,6 +235,7 @@ MissionBase::on_activation()
 	checkClimbRequired(_mission.current_seq);
 	set_mission_items();
 
+	_replay_index = _mission.current_seq;
```

Comment:
```suggestion
	_mission_activation_index = _mission.current_seq;
```
I always try to name variables as descriptive as possible.